mikkoo/worker.py

```
@@ -138,13 +138,16 @@ async def connect_to_postgres(self) -> bool:
-        return sql.SQL('SELECT * FROM {proc_name}({values})'.format(
-            proc_name=proc_name,
-            values=','.join('%s' for _a in range(0, len(args))))).as_string(
-                self.postgres)
-
-    async def callproc(self, proc_name: str, *args) \
-            -> typing.Optional[typing.Sequence[dict]]:
+        return sql.SQL(
+            'SELECT * FROM {proc_name}({values})'.format(
+                proc_name=proc_name,
+                values=','.join('%s' for _a in range(0, len(args))),
+            )
```

Ruff (0.15.6)

[error] Possible SQL injection vector through string-based query construction

(S608)